endorsement/storeapi.proto

```diff
@@ -5,10 +5,30 @@ option go_package = "github.com/veraison/endorsement";
 
 import "google/protobuf/struct.proto";
 import "result.proto";
-import "tokenformat.proto";
+
+enum AttestationFormat {
+	// UnknownFormat is used to indicate that the format of the attestation could
+	// not be established.
+	UNKNOWN_FORMAT = 0;
+
+	// PSA-IOT is the PSA attestation format (based on:
+	// https://developer.arm.com/architectures/architecture-security-features/platform-security)
+	PSA_IOT = 1;
+
+	// DICE is the attestation format based on the TCG DICE specification
+	// https://trustedcomputinggroup.org/wp-content/uploads/TCG_DICE_Attestation_Architecture_r22_02dec2020.pdf
+	DICE = 2;
+}
+
+enum TaType {
```

Comment:
I would spell the name out fully -- ``TrustAnchorType``; it's not that long, and is a lot clearer at a glance.